README.md

```diff
@@ -1,4 +1,7 @@
-![Build Status](http://storage.googleapis.com/java-runtimes-kokoro-build-badges/openjdk-runtime-master.png)
+![Build Status](http://storage.googleapis.com/java-runtimes-kokoro-build-badges/openjdk-runtime-master.png) 
+
+![Stability](https://img.shields.io/badge/gcr.io/google--appengine/openjdk:8-stable-green.svg)
+![Stability](https://img.shields.io/badge/gcr.io/google--appengine/openjdk:9-unstable-red.svg)
 
 # Google Cloud Platform OpenJDK Docker Image
 
```

Comment:
[in README.md](https://github.com/GoogleCloudPlatform/openjdk-runtime/pull/129/files#diff-04c6e90faac2675aa89e2176d2eec7d8R10): 
````
 This image is mirrored at both `launcher.gcr.io/google/openjdk8` and `gcr.io/google-appengine/openjdk`.
````
I think we'll need to update this for openjdk9 as well I guess? How do we differentiate between 8 and 9 in the google-appengine version?